infor-design/enterprise
# 5253 - Add Cards' single and multiple selection state

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR implements single and multi-selection of cards.

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/5253

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) Single Selection
- Go to http://localhost:4000/components/cards/example-single-select.html
- Click any of the cards
- It should be selectable and highlighted
- It should only select single card
- Test the UI across all themes and modes

2.) Multi-Selection
- Go to http://localhost:4000/components/cards/example-multi-select.html
- Compare to a single selection, multi-select should have checkboxes
- Click the cards, it should be selectable and the checkbox should be checked
- Clicking either the checkbox or the card itself should work the same
- Unselect the currently selected card/s
- It should unchecked and unhighlighted the card/s
- Test the UI across all themes and modes

**Included in this Pull Request**:
